### ISSUE TITLE
Is the <acqinfo> (acquisition information) field behaving as intended?

### ISSUE THREAD

**Issue by seanaery** [projectblacklight/arclight#1490]
This ticket was spawned out of #784 as it is still in need of verification.

### Current Behavior

The **Acquisition Information** field (`<acqinfo>` in EAD) currently behaves like none other in arclight. Is this intentional or should we change this? 
- When a component is being indexed, it always gets its owning collection's `<acqinfo>` value. It also gets it own `<acqinfo>` values (if present). [See traject rules here](https://github.com/projectblacklight/arclight/blob/main/lib/arclight/traject/ead2_component_config.rb#L272-L275).
- A component does not inherit any of its ancestor components' `<acqinfo>` values, only its top-level collection's `<acqinfo>`.
- This means on every single component page, the metadata includes `Acquisition information`. [Demo Example](https://arclight-demo.projectblacklight.org/catalog/umich-bhl-0312aspace_66733ea1d7a9837dfba9e48d32c4d61b). 
- There are no other fields that behave this way, where the component's value is inherited from the value at its collection level.

**Comment by gwiedeman** [comment]
In practice, archival systems have not inherited description like this. Instead the user is expected to infer that the upper level description applies. In a paper finding aid, a user would be expected to assume the collection level acquisition statement in the front matter applied to everything basically. Depending on the design of the system, this expectation may or may not be reasonable.

However, [DACS](https://saa-ts-dacs.github.io/dacs/06_part_I/02_chapter_01.html#requirements-for-multilevel-descriptions) _does_ say that every element must have [every required multilevel element](https://saa-ts-dacs.github.io/dacs/06_part_I/02_chapter_01.html#multilevel-required), but if its the same as the level above it should be omitted and the implicit inheritance applies. I think many archivists are not totally aware of this, and its kind of treated as a technicality.

With Arclight, we can probably manage this implicit inheritance better than archivists have in the past, as its not always clear to users. We're already doing that for access and use restrictions, it just displays in a separate section. This is a great example of better leveraging inheritance, as users shouldn't be expected to click to another page to see if the thing they are using is restricted.

I'm super interested in indexing parent description, like `<unittitle>`, `<scopecontent>`, or `<bioghist>` and such, but not displaying it, or displaying it _differently_, and mostly have it assist search results. So, for vague component titles like [Drawings](https://arclight-demo.projectblacklight.org/catalog/umich-bhl-0312aspace_5fea8810687f282d9f74bc325605a0d3), you might have more detailed description in that component Solr doc telling the context of the drawings that might assist search relevancy. But that is theoretical and I have no idea how it would work in practice. It would take a lot of search relevancy and UX testing and experimenting with users. `<acqinfo>` would not be my first choice for this.

Basically, if we are inheriting description like this, the immediate parent component's `<acqinfo>` (if its present) should apply over the collection `<acqinfo>`. And if it displays, it should probably be in another section ("Context" maybe?) so its clearer to the user that its inherited. But perhaps its best now for the current work cycle to just remove it, as archivists will totally not expect it there.

**Comment by mmmmcode** [comment]
@seanaery I second @gwiedeman 's comments - archivists won't expect collection-level <acqinfo> to appear at the component levels (but would expect component-level notes to display). It could even be misleading for users if there are multiple accessions listed at the collection level, but a folder/series/etc may have only come from one of those accessions.

**Comment by seanaery** [comment]
Thanks so much for your perspective, y'all! In that case, I will take this ticket, pull this into the current sprint, and just remove that indexing rule that's leading to the inherited component `acqinfo`. I'm sure we will have ample opportunities to explore other kinds of inheritance for discovery & display in future iterations.
